.github/workflows/ci.yml

```diff
@@ -28,13 +31,13 @@ jobs:
     - name: Check formatting
       run: cargo fmt -- --check
     - name: Documentation
-      run: cargo doc --workspace
+      run: cargo doc --workspace --exclude netcdf-src
     - name: Clippy
-      run: cargo clippy --workspace -- -D warnings
+      run: cargo clippy --workspace --exclude netcdf-src -- -D warnings
```

Comment:
@magnusuMET You can exclude `netcdf-src` from `default-members` in Cargo.toml and then all cargo commands will ignore it without you having to type `--exclude` everywhere :)